YACS-RCOS/yacs.n
# Updated summer2021.sh to account for the partial semesters

**Issue**

Note which issue number(s) this Pull Request closes. All pull requests should close an issue, if one does not exists, make one. This helps us plan and discuss before we decide on implementation

Closes #412 

**Test Procedure**

Show procedure to test functionality with a clear procedure

*Example:*
> 1. Load the update summer2021.csv into the tempDeploy
> 2. Ensure everything loads properly

**Additional Info**

This should create 3 summer semester options, full, session II and session III
